apps/api-server/migrations/074-create-notification-templates.js

```diff
@@ -0,0 +1,60 @@
+const { Sequelize } = require('sequelize');
+
+module.exports = {
+
+  async up ({ context: queryInterface }) {
+    queryInterface.createTable('notification_templates', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      projectId: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+      },
+      engine: {
+        type: Sequelize.ENUM('email', 'sms', 'carrier pigeon'),
```

Comment:
Ben echt benieuwd naar hoe carrier pigeon werkt xD

Reply:
Grapje. Hoezo wil je dat er uit hebben?

Reply:
Denk dat het handiger is als we dit inderdaad weg halen, dit lijkt me eerder testcode en als het niet echt gebruikt dient te worden dan kan het denk ik beter uit de codebase gehaald worden